infrastructure/versions.tf

```
@@ -0,0 +1,20 @@
+terraform {
+  required_version = "~> 1.5.3"
+
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "~> 5.12.0"
+    }
+  }
+
+  backend "s3" {
+    bucket = "master-the-legacy"
+    key    = "tf-states/infrastructure/terraform.tfstate"
+    region = "us-east-1"
+  }
+}
+
+provider "aws" {
+  # Configuration options
```

Optional 
```suggestion
   default_tags {
     tags = {
             project = var.project,
             environment = var.environment,
             owners = "minders"
     }
   }
```

Then those tags will be used in every resource as default merged to the specific in the resource